assets/src/edit-story/components/checklistCard/styles.js

```diff
@@ -24,11 +24,12 @@ import styled, { css } from 'styled-components';
  */
 import { Icons, THEME_CONSTANTS, Text } from '../../../design-system';
 import { focusableOutlineCSS } from '../../../design-system/theme/helpers';
+import { CARD_WIDTH } from '../helpCenter/navigator/constants';
 import { OverflowThumbnail } from '../thumbnail';
 import { GRID_TEMPLATE_AREA, GRID_VARIANT } from './constants';
 
 export const Wrapper = styled.div`
-  width: 272px;
+  width: ${CARD_WIDTH}px;
```

Comment:
```suggestion
  width: `${CARD_WIDTH}px`;
```

Reply:
Note that this is already a tagged template literal, so this won't work. It's correct as-is.

Reply:
I don't think this needs to be wrapped with back ticks because there are already backticks that wrap all of these styles 🤔